index.html

```diff
@@ -62,23 +62,28 @@ <h3>Favourite games</h3>
        
         <h3>Haiku</h3>
 
-            A world of dew,<br><br>
-            And within every dewdrop,<br><br>
-            A world of struggle.<br><br>
-        <a><cite>by <span class="author">Kobayashi Issa</span></cite></a>
+        <p>
+            A world of dew,<br>
+            And within every dewdrop,<br>
+            A world of struggle.<br>
+            <cite>by <span class="author">Kobayashi Issa</span></cite>
+        </p>
 
```

Comment:
Unnecessary `<a>`
Use only one `<br>` to separate lines in a poem
Missing `<p>`